packages/core/src/components/pill/pill.scss

```diff
@@ -15,7 +15,8 @@ $border-radius: 100px;
 $chip-height: 1.25rem;
 
 :host {
-  display: inline-flex;
+  display: flex;
```

Comment:
We cant remove inline-prefix here (https://www.w3schools.com/css/css_inline-block.asp), because this cause a line-break.